src/Riak/Api/Http.php

```diff
@@ -166,6 +167,9 @@ protected function buildPath()
             case 'Basho\Riak\Command\MapReduce\Fetch':
                 $this->path = sprintf('/%s', $this->config['mapred_prefix']);
                 break;
+            case 'Basho\Riak\Command\Indexes\Query':
+                $this->path = $this->createIndexQueryPath($this->command, $bucket);
```

Comment:
Passing the command to this method is unnecessary as the command is accessible from createIndexQueryPath and you already evaluated that it is the Indexes\Query command class.